client/src/app/components/Toolbar/Toolbar.jsx

```diff
@@ -16,28 +16,42 @@ import UserBar from './components/UserBar';
 import AppLogo from 'common/components/AppLogo';
 import CreationForm from 'modules/shopping-list/components/CreationForm';
 import { createNewShoppingList } from 'modules/shopping-list/model/actions';
+import { createCohort } from 'modules/cohort/model/actions';
 
 class Toolbar extends PureComponent {
   state = {
+    cohortFormVisbility: false,
     shoppingFormVisbility: false
   };
 
-  handleShoppingFormVisibilty = () => {
-    const { shoppingFormVisbility } = this.state;
-    this.setState({ shoppingFormVisbility: !shoppingFormVisbility });
+  handleFormVisibilty = isShoppingList => {
```

Comment:
should we check if this is list or cohort? Maybe we should show only one form - the one that was just clicked and hide all others? Clicking outside of form (if it is empty) should cliese the form popup.